hawkbit-repository/hawkbit-repository-jpa/src/test/java/org/eclipse/hawkbit/repository/jpa/ControllerManagementTest.java

```diff
@@ -645,22 +794,32 @@ public void findTargetVisibleMetaDataBySoftwareModuleId() {
 
     @Test
     @Description("Verify that controller registration does not result in a TargetPollEvent if feature is disabled")
-    @ExpectEvents({@Expect(type = TargetCreatedEvent.class, count = 1),
-            @Expect(type = TargetPollEvent.class, count = 0)})
+    @ExpectEvents({ @Expect(type = TargetCreatedEvent.class, count = 1),
+            @Expect(type = TargetPollEvent.class, count = 0) })
     public void targetPollEventNotSendIfDisabled() {
         repositoryProperties.setPublishTargetPollEvent(false);
         controllerManagement.findOrRegisterTargetIfItDoesNotExist("AA", LOCALHOST);
         repositoryProperties.setPublishTargetPollEvent(true);
     }
 
+    @Test
+    @Description("Verify that controller registration does not result in a TargetPollEvent if feature is disabled")
+    @ExpectEvents({ @Expect(type = TargetCreatedEvent.class, count = 1),
+            @Expect(type = TargetPollEvent.class, count = 0) })
+    public void targetPollEventNotSendIfDisabledWithName() {
```

Comment:
this test does not make any sense, because we are testing event filter here

Reply:
Removed.